docs/overview/support.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Support
+last_update:
+  date: 24 January 2024
+---
+
+There are many places that you can use to reach out for developer support, including:
+
+- Tezos Discord: https://discord.gg/yXaPy6s5Nr
+- Tezos sub-reddit: https://www.reddit.com/r/tezos/
+- Tezos Stack Exchange: https://tezos.stackexchange.com
+- Tezos Element community: https://riot.tzchat.org/
+- #tezos channel on Freenode
+
+For more social media links, see https://tezos.com/community/.
+
```

Comment:
It's a good idea to list the other documentation website. We can add more value by making it a documentation map, clearly defining the scope of each website, including the current one, to avoid the impression of useless duplication and create a clear mental model for developers:
```suggestion

The current website (https://docs.tezos.com/) is not exhaustive. It aims covering the main topics concerning Tezos developers and the main Tezos features, in the form of documentation and tutorials.
```